scrapi/harvesters/elife.py

```
@@ -0,0 +1,171 @@
+"""
+A harvester for metadata from eLife on Github for the SHARE project
+
+Sample API call:https://api.github.com/repos/elifesciences/elife-articles/commits?since=2015-12-22&until=2015-12-31&page=1&per_page=100
+"""
+
+from __future__ import unicode_literals
+
+import logging
+import datetime
+
+from lxml import etree
+import json
+
+from scrapi import requests
+from scrapi import settings
+from scrapi.base import XMLHarvester
+from scrapi.linter.document import RawDocument
+from scrapi.base.helpers import build_properties, compose, single_result, datetime_formatter
+
+logger = logging.getLogger(__name__)
+
+
+def title_parser(title):
+    parsed_title = ''.join(title)
+    return parsed_title
+
+
+def description_parser(description):
+    try:
+        if "DOI:" in description[-3]:
+            del description[-3:]
+    except:
```

Don't use bare except blocks, they can be pretty dangerous.